QuestionBot/QuestionBot/Model/QuestionMessageListener.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuestionBot.Model{
+
+    public class QuestionMessageListener : IMessageListener{
+
+        private IStore _questionDataStore;
+
+        public QuestionMessageListener(IStore store){
+            _questionDataStore = store;
+        }
+
+        public void ReceiveMessage(string input){
+            const string questionCommand = "/question";
```

Comment:
Since this constant is declared in a function, it is initialized every time the function is called. We can save a teeny amount of performance and a bit of readability by declaring it outside the function.